src/otx/core/config/data.py

```diff
@@ -81,6 +89,7 @@ class DataModuleConfig:
     test_subset: SubsetConfig
 
     tile_config: TilerConfig
+    vpm_config: VisualPromptingConfig
```

Comment:
```suggestion
    vpm_config: VisualPromptingConfig = VisualPromptingConfig()
```

Reply:
Is this working? If so please change this as well.
```python
tile_config: TilerConfig = TilerConfig()
```

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/2834/commits/5142fdef8dd643361b6bc9f0de8b6fdbb5886563.